pkg/term/term_linux_test.go

```diff
@@ -31,85 +32,87 @@ func newTempFile() (*os.File, error) {
 func TestGetWinsize(t *testing.T) {
 	tty, err := newTtyForTest(t)
 	defer tty.Close()
-	require.NoError(t, err)
+	assert.NilError(t, err)
 	winSize, err := GetWinsize(tty.Fd())
-	require.NoError(t, err)
-	require.NotNil(t, winSize)
-	require.NotNil(t, winSize.Height)
-	require.NotNil(t, winSize.Width)
```

Comment:
This is a great example of one of the many problems with testify. These asserts did nothing because `int` can never be nil, so the assert always passes, regardless of the value.  

Passing any non-nillable type to `Nil/NotNil` should always fail.